### ISSUE TITLE
Land after RTH - delayed landing.

### ISSUE THREAD

**Issue by PR31D** [iNavFlight/inav#6173]
## Current Behavior
As expected. On the craft returning home it immediately begins to land.

## Desired Behavior
On the craft returning to home, if it could loiter a set amount of time before starting to land. 

## Suggested Solution
Add in a variable delay where the craft will loiter before starting to land.

## Who does this impact? Who is this for?
This is mainly for fixed wing that dont crash well. I fly lion long range and I cant have my failsafe set to RTH and loiter as it could be up there loitering for a long time. However if im at short range and I've an issue with the link, the wing will start to land immediately on return to home. Not giving me any time to try sort the link out.

**Comment by PR31D** [comment]
👎

**Comment by MrD-RC** [comment]
This is an excellent idea for pilots who fly from a site unsuitable for auto landing. It will give time to try and fix problems, knowing that the model is safe above them.

**Comment by PR31D** [comment]
Hi Darren,

Thanks for the feedback.




________________________________

**Comment by MrMap** [comment]
Very good that this is merged into new versions.
Here is a video illustrating a CFIT ending of a test flight otherwise sussesful (ending with a FS test).
On a side note my fixedwing "landed" on cruise throttle, despite _set failsafe_throttle = 1000 (I will submit an issue on this).

Had this new landing delay functionality been in place I would not have to use so much glue.
That said I am painfully aware that this was all on me, not having thought it through properly (FS test procedure).
Enjoy the video!
https://youtu.be/9HkOczusHDI

**Comment by b14ckyy** [comment]
Should be no issue anymore with https://github.com/iNavFlight/inav/pull/9713
Additional features are planned for the near future, for example only autoland with defined landing spots but to avoid circle landing. Will keep this open for now.

**Comment by b14ckyy** [comment]
Oh here we go, its implemented. Closing https://github.com/iNavFlight/inav/blob/master/docs/Settings.md#nav_rth_fs_landing_delay
